backend/molgenis-emx2-sql/src/main/java/org/molgenis/emx2/sql/Migrations.java

```diff
@@ -0,0 +1,38 @@
+package org.molgenis.emx2.sql;
+
+import java.io.IOException;
+import org.molgenis.emx2.MolgenisException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class Migrations {
+  private static Logger logger = LoggerFactory.getLogger(Migrations.class);
+
+  public static void initOrMigrate(SqlDatabase db) {
+    int version = MetadataUtils.getVersion(db.getJooq());
```

Comment:
make sync or use concurrent lib ? to avoid race

Reply:
this function is only called once at startup of the server. So will only crash if you perfectly time the startup of two servers on the same postgresql instance. Parallel sessions threads do not run this method.